test/extensions/filters/udp/dns_filter/dns_filter_test.cc

```diff
@@ -400,17 +400,14 @@ TEST_F(DnsFilterTest, InvalidQuery) {
   query_ctx_ = response_parser_->createQueryContext(udp_response_, counters_);
   EXPECT_FALSE(query_ctx_->parse_status_);
 
-  EXPECT_EQ(DNS_RESPONSE_CODE_FORMAT_ERROR, response_parser_->getQueryResponseCode());
+  EXPECT_EQ(DNS_RESPONSE_CODE_FORMAT_ERROR, response_parser_->getQueryResponseCode(query_ctx_));
   EXPECT_EQ(0, query_ctx_->answers_.size());
 
   // Validate stats
   EXPECT_EQ(0, config_->stats().a_record_queries_.value());
   EXPECT_EQ(1, config_->stats().downstream_rx_invalid_queries_.value());
   EXPECT_TRUE(config_->stats().downstream_rx_bytes_.used());
   EXPECT_TRUE(config_->stats().downstream_tx_bytes_.used());
-
-  EXPECT_EQ(DNS_RESPONSE_CODE_FORMAT_ERROR, response_parser_->getQueryResponseCode());
```

Comment:
this was just unintended duplicate statements before, right?

Reply:
Correct